README.md

```diff
@@ -0,0 +1,237 @@
+
+You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+
+## Learn More <a name="learn-more"></a>
+
+You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+
+To learn React, check out the [React documentation](https://reactjs.org/).
+
+### Code Splitting
+<a name="code-splitting"></a>
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+
+### Analyzing the Bundle Size
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+
+### Making a Progressive Web App
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+
+### Advanced Configuration
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+
+### Deployment
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+
+### `npm run build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+<!-- AUTHOR -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **Alelign Ayana**
+- GitHub: [@atatm](https://github.com/atatm)
+- Twitter: [@alelign_ayana](https://twitter.com/@alelign_ayana)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/alelign-ayana/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Advanced styles**
+- [ ] **Testing with Jest**
+- [ ] **Responsive design in mobile version**
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank:
+- [Microverse](www.microverse.com)
+- [Nelson Sakwa in Behance](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
+- [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/)
```

Comment:
- [ **OPTIONAL** ] Good job adding the changes required by the previous reviewer. But I think it it would be better if you combine both sentences into one **sentence** as both of them talk about one person, and replace the last one like in the example below 
-Example:
```
- [Nelson Sakwa in Behance](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).
```